firmware/andes_firmware.ksy

```diff
@@ -0,0 +1,28 @@
+meta:
+  id: andes_firmware
+  endian: le
```

Comment:
Generally, there is minimal requirement for files in this repository to have `/meta/license` and it's highly preferred to have some metadata, like `/meta/title`, `/meta/application`, and `/doc`.

Could you please at least add a `license: XXX`, where `XXX` is a [valid SPDX license identifier](https://spdx.org/licenses/) for this work?